src/DotNetLightning.Core/Chain/KeysInterface.fs

```diff
@@ -57,8 +57,8 @@ type IKeysRepository =
     /// restarted with some stale data.
     abstract member GetChannelKeys: inbound:bool -> ChannelKeys
 
-    /// Must add signature to the PSBT *And* return the signature
-    abstract member GetSignatureFor: psbt: PSBT * pubKey: PubKey -> TransactionSignature * PSBT
+    /// Must add funding pub key signature for to the PSBT *And* return the signature
+    abstract member SignWithFundingPrivKey: psbt: PSBT -> TransactionSignature * PSBT
```

Comment:
"Making that explicit..." -> Sounds like this commit is a refactoring too (no change of behaviour), let's state it in the commit title

Reply:
Fixed.